code/lib/cli/src/detect.ts

```diff
@@ -158,71 +158,61 @@ export function detectPnp() {
   return pathExistsSync(join(process.cwd(), '.pnp.cjs'));
 }
 
-export function detectLanguage(packageJson?: PackageJson) {
+export async function detectLanguage(packageManager: JsPackageManager) {
   let language = SupportedLanguage.JAVASCRIPT;
 
-  // TODO: we may need to also detect whether a jsconfig.json file is present
-  // in a monorepo root directory
-  if (!packageJson || fs.existsSync('jsconfig.json')) {
+  if (fs.existsSync('jsconfig.json')) {
     return language;
   }
 
+  const typescriptVersion = await packageManager.getPackageVersion('typescript');
```

Comment:
I am not sure about this change. If the mono repository has Typescript installed, it does not mean, that the user would also like to use Typescript in Storybook